tidb-cloud/limited-sql-features.md

```diff
@@ -121,114 +121,110 @@ TiDB Cloud works with almost all workloads that TiDB supports, but there are som
-| `tidb_stmt_summary_max_stmt_count` | No limitation | Read-only [^11] |
-| `tidb_stmt_summary_refresh_interval` | No limitation | Read-only [^11] |
-| `tidb_sysproc_scan_concurrency` | No limitation | Read-only [^11] |
+| `tidb_stats_load_sync_wait` | No limitation | Read-only [^10] |
+| `tidb_stmt_summary_history_size` | No limitation | Read-only [^10] |
+| `tidb_stmt_summary_internal_query` | No limitation | Read-only [^10] |
+| `tidb_stmt_summary_max_sql_length` | No limitation | Read-only [^10] |
+| `tidb_stmt_summary_max_stmt_count` | No limitation | Read-only [^10] |
+| `tidb_stmt_summary_refresh_interval` | No limitation | Read-only [^10] |
+| `tidb_sysproc_scan_concurrency` | No limitation | Read-only [^10] |
 | `tidb_top_sql_max_meta_count` | Not supported [^4] | Not supported [^4] |
 | `tidb_top_sql_max_time_series_count` | Not supported [^4] | Not supported [^4] |
-| `tidb_tso_client_batch_max_wait_time` | No limitation | Read-only [^11] |
-| `tidb_ttl_delete_batch_size` | No limitation | Read-only [^9] |
-| `tidb_ttl_delete_rate_limit` | No limitation | Read-only [^9] |
-| `tidb_ttl_delete_worker_count` | No limitation | Read-only [^9] |
-| `tidb_ttl_job_enable` | No limitation | Read-only [^9] |
-| `tidb_ttl_job_schedule_window_end_time` | No limitation | Read-only [^9] |
-| `tidb_ttl_job_schedule_window_start_time` | No limitation | Read-only [^9] |
-| `tidb_ttl_running_tasks` | No limitation | Read-only [^9] |
-| `tidb_ttl_scan_batch_size` | No limitation | Read-only [^9] |
-| `tidb_ttl_scan_worker_count` | No limitation | Read-only [^9] |
-| `tidb_txn_mode` | No limitation | Read-only [^11] |
-| `tidb_wait_split_region_finish` | No limitation | Read-only [^11] |
-| `tidb_wait_split_region_timeout` | No limitation | Read-only [^11] |
-| `txn_scope` | No limitation | Read-only [^11] |
-| `validate_password.enable` | No limitation | Always enabled [^10] |
-| `validate_password.length` | No limitation | At least `8` [^10] |
-| `validate_password.mixed_case_count` | No limitation | At least `1` [^10] |
-| `validate_password.number_count` | No limitation | At least `1` [^10] |
-| `validate_password.policy` | No limitation | Can only be `MEDIUM` or `STRONG` [^10] |
-| `validate_password.special_char_count` | No limitation | At least `1` [^10] |
-| `wait_timeout` | No limitation | Read-only [^11] |
+| `tidb_tso_client_batch_max_wait_time` | No limitation | Read-only [^10] |
+| `tidb_ttl_delete_batch_size` | No limitation | Read-only [^10] |
+| `tidb_ttl_delete_rate_limit` | No limitation | Read-only [^10] |
+| `tidb_ttl_delete_worker_count` | No limitation | Read-only [^10] |
+| `tidb_ttl_job_schedule_window_end_time` | No limitation | Read-only [^10] |
+| `tidb_ttl_job_schedule_window_start_time` | No limitation | Read-only [^10] |
+| `tidb_ttl_running_tasks` | No limitation | Read-only [^10] |
+| `tidb_ttl_scan_batch_size` | No limitation | Read-only [^10] |
+| `tidb_ttl_scan_worker_count` | No limitation | Read-only [^10] |
```

Comment:
for L203~L210, are these ttl variables still ready-only?

Reply:
yes，still read-only.